pkg/api/handlers/util.go

```diff
@@ -2,6 +2,9 @@ package handlers
 
 import (
 	"fmt"
+
+	"github.com/pkg/errors"
```

Comment:
is the use of this package on purpose? I have a feeling this crept in `goimports` and the standard `errors` package should be used here.

Reply:
Fixed, thanks 👍